cmd/main.go

```diff
@@ -254,9 +258,12 @@ func getDPAConfiguration(restConfig *rest.Config, oadpNamespace string) (v1alpha
 			if nonAdmin.BackupSyncPeriod != nil {
 				dpaConfiguration.BackupSyncPeriod.Duration = nonAdmin.BackupSyncPeriod.Duration
 			}
+			if dpa.Spec.Configuration.Velero.Args != nil && dpa.Spec.Configuration.Velero.Args.BackupSyncPeriod != nil {
+				defaultSyncPeriod = dpa.Spec.Configuration.Velero.Args.BackupSyncPeriod
```

Comment:
In case this is not met, we may have nil point in the caller of this function, because we will have dpaConfiguration, but defaultSyncPeriod was never initialized from line 231.

Reply:
I think this is ok

I want to it to be nil to check it here https://github.com/migtools/oadp-non-admin/pull/223/files#diff-de455907ed1c5413fa6248ebea483af559585f8f6277a7c8b370813b9bb254f9R232

or does this break usage?

Reply:
I think it may break us if the DPA is set without default BSL (so no BSL at all in the cluster and DPA reconciled), then we won't ValidateBSL properly, need to test this scenario.